packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
For me, adding https://github.com/OCamlPro/alt-ergo as a dev-repo for Alt-Ergo is wrong (even if it seems to be the case from an external point of view).

The other thing is that this addition forces the additional dependency "conf-autoconf", which is not necessary without dev-repo.